audio/audio_state.cc

```diff
@@ -124,6 +124,17 @@ void AudioState::RemoveSendingStream(webrtc::AudioSendStream* stream) {
   UpdateAudioTransportWithSendingStreams();
   if (sending_streams_.empty()) {
     config_.audio_device_module->StopRecording();
+#if defined(WEBRTC_IOS)
+    auto* adm = config_.audio_device_module.get();
+    if (adm->Playing()) {
+      adm->StopPlayout();
+      if (adm->InitPlayout() == 0) {
```

Comment:
what does this mean? we are starting playout after stopping it?

Reply:
Through repeated tests, when the mode is changed to playback, the RTCAudioSession must be reset to stop the capture of the microphone.

Reply:
I'll add a note describing the intent of this PR.

Reply:
got it, so it'd have to be restarted after Playout is stopped?

Reply:
If playout is stopped, we should not need to restart it， we only need to deal with `adm->StopRecording()`